servant-io-streams/servant-io-streams.cabal

```diff
@@ -0,0 +1,57 @@
+cabal-version:       2.2
+name:                servant-io-streams
+version:             0.1
+
+synopsis:            Servant Stream support for io-streams
+category:            Servant, Web, io-streams
+description:         Servant Stream support for io-streams.
+  .
+  Provides 'ToSourceIO' and 'FromSourceIO' instances for 'InputStream'.
+
+homepage:            http://docs.servant.dev/
+bug-reports:         http://github.com/haskell-servant/servant/issues
+license:             BSD-3-Clause
+license-file:        LICENSE
+author:              Servant Contributors
+maintainer:          haskell-servant-maintainers@googlegroups.com
+copyright:           2023 Servant Contributors
+build-type:          Simple
+tested-with: GHC ==8.6.5 || ==8.8.4 || ==8.10.2
+
+extra-source-files:
+  CHANGELOG.md
+
+source-repository head
+  type: git
+  location: http://github.com/haskell-servant/servant.git
+
+library
+  exposed-modules:     Servant.IO.Streams
+  build-depends:
+      base        >=4.9  && <5
+    , io-streams ^>=1.5
+    , servant     >=0.15 && <0.20
```

Comment:
These bounds are not correct, will need to update to whatever the new major version will be that includes #1661